slices/fonts-noto-mono.yaml

```diff
@@ -0,0 +1,19 @@
+package: fonts-noto-mono
+
+essential:
+  - fonts-noto-mono_copyright
+
+slices:
+  fonts:
```

Comment:
Same as above: shouldn't the `fonts-noto-mono_config` be listed as an essential here?

Reply:
Any thoughts?

_(Please let me know if this was previously requested to be secluded.)_